bridge/qb/server/main.lua

```diff
@@ -20,6 +21,8 @@ qbCoreCompat.UsableItems = QBX.UsableItems
 qbCoreCompat.Functions = require 'bridge.qb.server.functions'
 qbCoreCompat.Commands = require 'bridge.qb.server.commands'
 
+---@diagnostic disable: deprecated
+
 ---@deprecated Call lib.print.debug() instead
```

Comment:
Curious what is triggering warnings about this line?

Reply:
All the `qbCoreCompat.Functions...` and `qbCoreCompat` callbacks in the code below. I just figured it'd be better to toss the diagnostic for it all instead of going line by line for each of them.